ReadMe.md

```diff
@@ -162,6 +162,18 @@ console.log(colors.error("this is an error"));
 
 // outputs yellow text
 console.log(colors.warn("this is a warning"));
+
+```
+
+You can also combine them:
+
+```javascript
+colors.setTheme({
+  link: ['underline', 'blue']
+});
+
+// outputs underlined blue text
+console.log(colors.info('Listening on ') + colors.link('http://0.0.0.0:' + port)  );
```

Comment:
`info` and `colors` is defined in the code that is above in the README and port was pulled from some working code. I didn't think it was at all unclear (you have another example that doesn't include `colors=require('colors')`) but feel free to write it however you like. I can see that maybe including a `port` variable that isn't defined is less clear than `console.log(colors.info('Listening on ') + colors.link('http://0.0.0.0:8080')  );`. My issue was with "won't actually run correctly" which seemed to also happen somewhere in this process, when the `String.prototype` stuff was pulled out and the `colors` API wasn't being properly updated.